html/index.html

```diff
@@ -144,45 +144,50 @@ <h2 class="my-0 fw-semibold">Nautobot Apps</h2>
                                 Development of each App.</p>
                         </div>
                         <ul class="list-group list-group-flush">
-                            <li class="list-group-item"><a href="https://docs.nautobot.com/projects/core/en/stable/apps/"
-                                    class="w-100 btn btn-lg btn-warning" target="_blank">To the App Docs!</a>
+                            <li class="list-group-item"><a href="https://www.networktocode.com/nautobot/apps/"
```

Comment:
This is a docs site and we should be linking to docs - I don't agree with this link change here. The main button sending ppl to the main website (that doesn't link to docs) is not the way. The https://docs.nautobot.com/projects/core/en/stable/apps/ page links to the ecosystem page right at the top if people want to discover more.